### ISSUE TITLE
Cannot merge because "Unexpected token function" issue

### ISSUE THREAD

**Issue by vitali2y** [RyuuGan/sol-merger#6]
Hi,

After [standard installation](https://github.com/RyuuGan/sol-merger#usage) (node v6.9.4, npm 4.1.1) had encountered next issue:
```
vit@lea ~/test $ ./node_modules/.bin/sol-merger './contracts/*.sol' ./build
/home/vit/test/node_modules/sol-merger/bin/mergeFile.js:27
async function execute() {
      ^^^^^^^^
SyntaxError: Unexpected token function
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
vit@lea ~/test $
```
Tnx.

**Comment by RyuuGan** [comment]
As far as I can see, the only thing why it is not working is that your `node` version is not supporting `async/await` pattern, since it is available from node `7.6.0`. 

So try upgrading node to version `8.x.x LTS` and the error should go away.

Best regards,
Valery Aligorsky

**Comment by vitali2y** [comment]
Yep, works as a charm with `node v8.8.1` & `npm 5.5.1` (how about to upgrade "`engines`" of "`package.json`"?).
Tnx.
